### DISCUSSION TITLE
Hello guys, how does coder launch vscode web?

### DISCUSSION THREAD

**Discussion by Pythoneryeah** [coder/coder#8311] [General] [answered]
Hello guys, how does coder launch vscode web?

 How is the code implemented?

**Comment by matifali** [comment, marked as answer]
Coder has a concept of `coder-app` where you can run and launch any web-based IDE or app. It acts a reverse proxy for the app running in your workspace.
Some examples of Coder apps are.

1. VS Code Web / code-server
2. Jupyter Notebook/Lab
3. RStudio Server
4. practically any web-based application.

**Reply by Pythoneryeah** [reply]
Thank you very much. I'll try
